doc/man7/RAND_DRBG.pod

```diff
@@ -264,6 +264,14 @@ from the trusted entropy sources.
 
 =back
 
+NOTE: Manual reseeding is *not allowed* in FIPS mode, because
+NIST SP-800-90A mandates that entropy *shall not* be provided by the
+consuming application, neither for instantiation, nor for reseeding.
+[NIST SP 800-90Ar1, Sections 9.1 and 9.2]. For that reason the B<randomness>
+argument is ignored and the random bytes provided by the L<RAND_add(3)> and
+L<RAND_seed(3)> calls are treated as additional data. (In other words, the
```

Comment:
I don't think the parenthesized sentence is good.  Either remove the parentheses or remove the whole thing (I don't think it adds much to the paragraph).

Reply:
I removed the sentence.